src/Illuminate/Support/Number.php

```diff
@@ -367,6 +367,30 @@ public static function defaultCurrency()
         return static::$currency;
     }
 
+    /**
+     * Generate a random number of the given length.
+     *
+     * @param  int  $length
+     * @return int
+     */
+    public static function random(int $length = 6): int
+    {
+        $maxLength = strlen((string) PHP_INT_MAX);
+
+        if ($length < 1 || $length > $maxLength) {
+            return 0;
```

Comment:
Is `0` helpful here? If I can't generate a _random_ integer, I'd expect to get an exception thrown